.ci/linters/c/omp_set_num_threads_linter.R

```diff
@@ -0,0 +1,13 @@
+
+# Ensure no calls to omp_set_num_threads() [to avoid affecting other packages and base R]
+# Only comments referring to it should be in openmp-utils.c
+omp_set_num_threads_linter = function(c_file) {
+  # strip comments, we only care if the function appears in actual code.
+  processed_lines = system2("gcc", c("-fpreprocessed", "-E", "-xc", "-", c_file), stdout=TRUE, stderr=FALSE)
```

Comment:
```suggestion
  processed_lines = system2("gcc", c("-fpreprocessed", "-E", "-xc", c_file), stdout=TRUE, stderr=FALSE)
```
I guess the empty option is a typo?

Reply:
Ah, that is for using `stdin=c_file`, I was trying out `stdin=lines`, which didn't work, `-xc -` is leftover from that. It's easiest to just use the filename directly.